techniques/failures/F85.html

```diff
@@ -23,14 +24,14 @@ <h3>Setting focus to the document after dismissing a menu embedded on the page</
                <p>Activate the trigger control via the keyboard.</p>
                <ul>
                   <li>Check whether focus is in the menu or dialog.</li>
-                  <li>Check whether advancing the focus in the sequential navigation order puts focus in the menu or dialog.</li>
+                  <li>Check whether moving the focus forwards once in the sequential navigation order puts focus in the menu or dialog.</li>
```

Comment:
Thinking of dialogs with no interactive content apart from the close icon. When focus is set here on opening, it is often not easy to see since there is no other focus position to go to - in effect you can either press enter or ESC to close. In these situations the second check would not work. Not sure if this edge case (not too infrequent though) needs to be covered...

Reply:
the "check" may not necessarily be a visual check. an auditor may also just check `document.activeElement`.